tests/10apidoc/09synced.pl

```diff
@@ -205,7 +205,34 @@ sub await_sync_presence_contains {
 }
 
 
-push @EXPORT, qw( assert_room_members assert_state_room_members_matches );
+push @EXPORT, qw( assert_room_members assert_state_room_members_matches assert_room_state);
```

Comment:
nice to have: generally it seems helpful to have a `push @EXPORT` next to each func being exported, rather than one big one at the top

Reply:
the codebase is very inconsistent on this, but have moved them.